cmd/find-main.go

```diff
@@ -67,17 +70,13 @@ var (
 			Name:  "smaller",
 			Usage: "Match all objects smaller than specified size in units (see UNITS)",
 		},
-		cli.StringFlag{
+		cli.UintFlag{
```

Comment:
Can we have the flags alphabetically sorted  and in the usage, for the end-user's readability, below, line#97?
Can we have the flags alphabetically sorted in the usage below, line#97, for end-user readability purposes, and here just for easy debugging purposes?